UI/setup/template_info.html

```diff
@@ -8,6 +8,11 @@ <h2><?lsmb text('Database Management Console') ?></h2>
 <form data-dojo-type="lsmb/SimpleForm" action="setup.pl" method="post">
 <div class="listtop"><?lsmb text('Select Templates to Load') ?></div>
 <?lsmb
+INCLUDE input element_data = {
```

Comment:
Why do you want to retain the country of the CoA? The CoA has already been loaded...